src/gam.py

```diff
@@ -7817,6 +7817,9 @@ def doCreateResoldCustomer():
 def _getValueFromOAuth(field, credentials=None):
   if not credentials:
     credentials = auth.get_admin_credentials()
+  if credentials.expired:
```

Comment:
This should really be handled in Credentials#get_token_value, itself. Otherwise, other callers are going to experience the same thing. I can draw up a PR with the fix